tests/smoke/events.js

```diff
@@ -49,7 +49,7 @@
 			bot.setHtmlWithSelection(starterHtml);
 
 			resumeAfter(editor, 'spellCheckComplete', function () {
-				observer.assert(["spellCheckComplete", "startRender", "startCheckWordsAjax", "startScanWords"])
+				observer.assert(["startScanWords", "startCheckWordsAjax", "startRender", "spellCheckComplete"])
```

Comment:
it gets a bit weird in the next one because some of them are more async than others.  but we'll get there soon...